src/app/teamperiods/teamperiods.component.spec.ts

```diff
@@ -51,4 +52,11 @@ describe('TeamPeriodsComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should copy people list correctly', () => {
+    const people = [];
+    people.push(new Person('alice', 'Alice Atkins', 'LON', 5),
```

Comment:
You can just use an array literal here, i.e. `const people = [new Person(...), new Person(...)]`. There is no need to start with an empty list and then push.